admin/routes/questions.js

```diff
@@ -0,0 +1,7 @@
+const express = require('express')
+const router = express.Router()
+const { getQuestions } = require('../controllers/questions')
+
+router.post('/', (req, res) => getQuestions(req, res))
```

Comment:
FYI - i have removed Jon and Paula as reviewers for now, until my requested changes have been made.  Please add them back in for a final review once the API is as per spec.